pkg/rootless/rootless_linux.c

```diff
@@ -471,7 +471,7 @@ create_pause_process (const char *pause_pid_file_path, char **argv)
             close (fd);
 
           setenv ("_PODMAN_PAUSE", "1", 1);
-          execlp (argv[0], NULL);
+          execlp (argv[0], "", NULL);
```

Comment:
this will be mapped to argv[0], since we are touching it, could we use `execlp (argv[0], argv[0], NULL);`?

Reply:
Yes, I will change it as suggested.